test/Makefile

```diff
@@ -17,6 +17,18 @@ RUNTEST=r2r -L -o results.json
 r2r-tests: bins
 	${RUNTEST}
 
+8051: bins
+	radare2 -a 8051 -m 0x8000 -e "scr.color=0" -qf -c "pI" ./bins/8051/MP_8192E_FW_NIC.bin > MP_8192E_FW_NIC.asm
```

Comment:
make clean should remove this .asm file

Reply:
Nice catch. I decided to put the .asm file in the ./tmp folder to delete it.